pkg/depository/get.go

```diff
@@ -17,17 +17,144 @@ limitations under the License.
 package depository
 
 import (
+	"crypto/tls"
+	"encoding/json"
 	"fmt"
+	"io"
+	"net/http"
+	"net/url"
 
+	"github.com/bestchains/bc-cli/pkg/common"
+	"github.com/bestchains/bc-cli/pkg/printer"
 	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 )
 
-func NewGetDepositoryCmd() *cobra.Command {
-	return &cobra.Command{
-		Use: "depository [NAME] [args]",
+func ConstructQuery(cmd *cobra.Command) string {
+	query := url.Values{}
+	from, _ := cmd.Flags().GetInt("from")
+	if from != 0 {
+		query.Add("from", fmt.Sprintf("%d", from))
+	}
+	size, _ := cmd.Flags().GetInt("size")
+	if size != 0 {
+		query.Add("size", fmt.Sprintf("%d", size))
+	}
+	kid, _ := cmd.Flags().GetString("kid")
+	if kid != "" {
+		query.Add("kid", kid)
+	}
+	name, _ := cmd.Flags().GetString("name")
+	if name != "" {
+		query.Add("name", name)
+	}
+	contentName, _ := cmd.Flags().GetString("contentName")
+	if contentName != "" {
+		query.Add("contentName", contentName)
+	}
+
+	return fmt.Sprintf("%s?%s", common.ListDepository, query.Encode())
+}
+
+var headers = []string{"index", "kid", "platform", "operator", "owner", "blockNumber", "time"}
```

Comment:
Make first charater upper case ? 
like 
```
Index Kid Platform Operator
```
At least, we should display in upper case